### ISSUE TITLE
Describe table endpoint

### ISSUE THREAD

**Issue by avelino** [prest/prest#281]
Method: `GET`
Endpoint: `/describe/DATABASE/SCHEMA/TABLE`
SQL to Postgres adapter:
```sql
SELECT *
FROM information_schema.columns
WHERE table_schema = 'SCHEMA'
AND table_name   = 'TABLE'
```

Return in JSON (by example):
```json
{
  "table_catalog":"DATABASE",
  "table_schema":"SCHEMA",
  "table_name":"TABLE",
  "column_name":"code",
  "ordinal_position":1,
  "column_default":null,
  "is_nullable":"NO",
  "data_type":"integer",
  "character_maximum_length":null,
  "character_octet_length":null,
  "numeric_precision":32,
  "numeric_precision_radix":2,
  "numeric_scale":0,
  "datetime_precision":null,
  "interval_type":null,
  "interval_precision":null,
  "character_set_catalog":null,
  "character_set_schema":null,
  "character_set_name":null,
  "collation_catalog":null,
  "collation_schema":null,
  "collation_name":null,
  "domain_catalog":null,
  "domain_schema":null,
  "domain_name":null,
  "udt_catalog":"DATABASE",
  "udt_schema":"pg_catalog",
  "udt_name":"int4",
  "scope_catalog":null,
  "scope_schema":null,
  "scope_name":null,
  "maximum_cardinality":null,
  "dtd_identifier":"1",
  "is_self_referencing":"NO",
  "is_identity":"NO",
  "identity_generation":null,
  "identity_start":null,
  "identity_increment":null,
  "identity_maximum":null,
  "identity_minimum":null,
  "identity_cycle":"NO",
  "is_generated":"NEVER",
  "generation_expression":null,
  "is_updatable":"YES"
}
```

**Comment by fabriziomello** [comment]
+1 for idea and -1 do use "information_schema".

I worry because "information_schema" implementation in PostgreSQL have performance issues mostly in database and with a high number of objects.

And I'm not totally convinced if we need return all this information. Some database have their own implementation of DESCRIBE command (i.e MySQL) an de maybe we need to define our return interfaces for our describe endpoint instead of simple return a query result.

**Comment by avelino** [comment]
MySQL describe fields:
- Field
- Type
- Null
- Key
- Default
- Extra

**Comment by avelino** [comment]
fixed in #410
